app/src/components/modals/EditNoteModal.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  Modal,
+  Pressable,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import {useState} from 'react';
+import {NoteStructureWithMatchNumber} from '../../database/Notes';
+import Svg, {Path} from 'react-native-svg';
+import {supabase} from '../../lib/supabase';
+
+export const EditNoteModal = ({
```

Comment:
Consider adding ability to delete notes from this modal.

Reply:
I don't think you should be able to delete notes - You can't delete a scout report, same goes for notes